package/gluon-status-page/src/build.js

```diff
@@ -1,6 +1,7 @@
 ({
   paths: {
-    "bacon": "../Bacon"
+    "bacon": "../Bacon",
+    "eventsource": "../EventSource"
```

Comment:
thanks for the info. I'm not familiar with the build scripts of gluon. @ecsv told me to introduce a patch in main gluon repo. I will change my request to the hints you made. We should support all browsers with or without a polyfill. I use the Chrome browser mainly but Edge is the common browser on Windows 10 and I think a handful of users will use this platform. I don't want to start a discussion about used platforms or browsers, but Edge is the main browser for Windows.

just wait for a new change from me before accept the pull request. thx